ecllibrary/std/Date.ecl

```diff
@@ -1159,9 +1159,9 @@ END;
  */
 
 EXPORT BOOLEAN IsValidTime(Time_t time) := FUNCTION
-    hourInBounds := (Hour(time) BETWEEN 0 AND 23);
```

Comment:
Not really related - I was originally testing with all warnings as errors too, and this was giving warnings in generated code about a redundant compare of an unsigned >= 0...